packages/core/src/utils/public.ts

```diff
@@ -290,3 +290,15 @@ export function getOsInfo() {
     });
   });
 }
+
+/**
+ * shuffle array
+ * @param array
```

Comment:
Could we have more documentation for this method?

Reply:
done